### DISCUSSION TITLE
How can I split an existing terminal?

### DISCUSSION THREAD

**Discussion by thiagodebastos** [folke/snacks.nvim#29] [Q&A] [answered]
I notice that in the screenshot for [terminal docs](https://github.com/folke/snacks.nvim/blob/main/docs/terminal.md) the terminal is split in two. How can I achieve this?

**Comment by folke** [comment, marked as answer]
Use a `count` before pressing your keymap that opens the terminal

**Reply by thiagodebastos** [reply]
Much appreciated @folke ! 🙏

**Reply by xian-wen** [reply]
Hi, when I use `2<C-/>` to open the second terminal, it no longer opens the terminal in the root directory, as shown in the [terminal doc](https://github.com/folke/snacks.nvim/blob/main/docs/terminal.md#-terminal) . How can I solve this?
![image](https://github.com/user-attachments/assets/6704f957-ecb5-4685-9a6f-27514f2c55c9)

**Reply by xian-wen** [reply]
Solved by https://github.com/folke/snacks.nvim/issues/385.
